src/user-guide/running-workflows.rst

```diff
@@ -1295,57 +1295,57 @@ Disaster Recovery
 If a run directory gets deleted or corrupted, the options for recovery
 are:
 
-- restore the run directory from back-up, and restart the suite
+- restore the run directory from back-up, and restart the workflow
 - re-install from source, and warm start from the beginning of the
   current cycle point
 
-A warm start (see :ref:`Warm Start`) does not need the suite database, but it
+A warm start (see :ref:`Warm Start`) does not need the workflow database, but it
 could re-run a significant number of tasks that had already completed.
 
-To restart the suite, the critical Cylc files that must be restored are:
+To restart the workflow, the critical Cylc files that must be restored are:
 
 .. code-block:: sub
 
-   # On the suite host:
-   ~/cylc-run/SUITE-NAME/
-       flow.cylc   # live suite configuration (located here in Rose suites)
-       log/db  # public suite DB (can just be a copy of the private DB)
-       log/rose-suite-run.conf  # (needed to restart a Rose suite)
-       .service/db  # private suite DB
-       .service/source -> PATH-TO-SUITE-DIR  # symlink to live suite directory
+   # On the workflow host:
+   ~/cylc-run/WORKFLOW-NAME/
+       flow.cylc   # live workflow configuration (located here in Rose workflows)
```

Comment:
Should it remain as "Rose suites"?

Reply:
The parenthesized bit can be deleted.

Reply:
(Done; but this section needs wider updates since `rose suite-run` migration).